data/framework/level1.ui

```diff
@@ -305,6 +286,30 @@
                 <property name="width">4</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkComboBox" id="fontChooser">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <child>
+                  <object class="GtkCellRendererText" id="fontRenderer">
+                    <property name="cell_background_gdk">#000000000000</property>
+                    <property name="cell_background_rgba">rgba(0,0,0,0)</property>
+                    <property name="xpad">2</property>
+                    <property name="ypad">2</property>
+                    <property name="xalign">0</property>
+                    <property name="font">Normal</property>
+                    <property name="foreground_gdk">#000000000000</property>
+                    <property name="background_rgba">rgba(0,0,0,0)</property>
+                    <property name="foreground_rgba">rgba(0,0,0,0)</property>
```

Comment:
I don't think these colors should be hardcoded here. This has the side-effect that when the font entry is selected from the combobox menu, it appears black-on-blue, which is hard to read.

Reply:
OK, that worked. I guess I needed to disable the cell renderer properties instead of leaving them at their defaults